### DISCUSSION TITLE
Grouping the table data

### DISCUSSION THREAD

**Discussion by Max1992** [jxlsteam/jxls#285] [Q&A]
Hello, 
I'm looking for a mechanism to make a grouping as i described on my example

![image](https://github.com/jxlsteam/jxls/assets/3974287/e94b0a7a-4622-485b-8ef6-0681db8a479c)

As you can see there are some kind of two grouping level
1. by move's name
2. by seasons

Also we have different column at each group's level.
Relese date is appliable only for 1'st level group.
First/Last aired are only for 2'nd level group.

The question is: is it possible to make this kind of report by using jx:* template?

Thanks.

**Comment by gastendonk** [comment]
closing old item
